wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUDataColumnInfo.java

```diff
@@ -26,11 +30,22 @@
      * 
      * @param base
      */
-    public DFUDataColumnInfo(DFUDataColumn base)
+    public DFUDataColumnInfo(org.hpccsystems.ws.client.gen.wsdfu.v1_36.DFUDataColumn base)
```

Comment:
I'm a little concerned about this... we can't create a DFUDataColumnInfo based on post wsdfu1.36 DFUDataColumns?